crowbar_framework/app/models/backup.rb

```diff
@@ -159,14 +159,19 @@ def save_archive
       return false
     end
 
+    if data.join("meta.yml").exist?
+      meta = YAML.load_file(data.join("meta.yml"))
```

Comment:
Beside that I would avoid this nesting level. Maybe something like this is better:

```ruby
unless data.join("meta.yml").exist?
  errors.add(:file_content, I18n.t("backups.index.meta_missing"))
  return false
end

begin
  meta = YAML.load_file(
    data.join("meta.yml")
  )
rescue YAML::ParseError
  errors.add(:file_content, I18n.t("backups.index.meta_invlaid"))
  return false
end

self.version = meta["version"]
self.size = path.size
self.created_at = Time.zone.parse(meta["created_at"])

...
```